src/modules/feature-modules/innovator/pages/innovation/how-to-proceed/how-to-proceed.component.ts

```diff
@@ -91,31 +83,16 @@ export class PageInnovationHowToProceedComponent extends CoreComponent {
     }
 
     switch (this.form.get('action')?.value) {
-      case FormFieldActionsEnum.ARCHIVE:
-        this.redirectTo(`${this.baseUrl}/how-to-proceed/archive`, {
-          action: FormFieldActionsEnum.ARCHIVE // update this to have the new available actions in the flow
-        });
-        break;
-
-      case FormFieldActionsEnum.DELETE_ACCOUNT:
-        this.redirectTo('/innovator/account/manage-account/delete', {
-          action: FormFieldActionsEnum.DELETE_ACCOUNT
-        });
+      case FormFieldActionsEnum.NEED_MORE_SUPPORT_NOW:
+        this.redirectTo(`/innovator/innovations/${this.innovationId}/how-to-proceed/recommend-needs-reassessment`);
         break;
-
-      case FormFieldActionsEnum.NEEDS_REASSESSMENT:
-        this.redirectTo(`${this.baseUrl}/how-to-proceed/needs-reassessment-send`, {
-          action: FormFieldActionsEnum.NEEDS_REASSESSMENT
-        });
-        break;
-
-      case FormFieldActionsEnum.NO_ACTION:
-      default:
-        this.setRedirectAlertSuccess('There is no active support for your innovation', {
-          message:
-            'You can decide whether to submit your innovation for a needs reassessment, archive your innovation or delete your account at any time.'
+      case FormFieldActionsEnum.WILL_DEVELOP_AND_COME_BACK:
+      case FormFieldActionsEnum.HAVE_ALL_I_NEED:
+      case FormFieldActionsEnum.DECIDED_NOT_TO_PURSUE:
+      case FormFieldActionsEnum.INNOVATION_IS_ALREADY_LIVE:
+        this.redirectTo(`/innovator/innovations/${this.innovationId}/how-to-proceed/archive`, {
+          ...(this.form.get('action')?.value && { action: this.form.get('action')!.value })
```

Comment:
This 'action' value check is not necessary since when using the switch case, the value would be one of those listed